pkg/juicefs/mount/pod_mount.go

```diff
@@ -410,7 +417,7 @@ func (p *PodMount) waitUtilJobCompleted(ctx context.Context, jobName string) err
 			"job-name": jobName,
 		},
 	}, nil)
-	if err != nil || len(pods) != 1 {
+	if err != nil || len(pods) == 0 {
```

Comment:
maybe `len(pods) != 0`?

Reply:
len(pods) == 0 means no pod of job is created, it's abnormal and need to return error.

Reply:
Ok